### PR TITLE
dd SetupWizard.licence command (v5) and is v4 compatibility

### PR DESCRIPTION
**Pull request by Tutul-**
+ custom command for string only licence (set by scriptor)

### CODE REVIEW

**Comment by Tutul-** [comment]
I just see your comment about File() object, correct it tomorrow, after reading your comment about these work :)

File: src/main/java/com/playonlinux/framework/SetupWizard.java [hunk 1]
```diff
@@ -103,6 +109,47 @@ public void execute(Message message) {
...
     }
+
+    /**
+     * Show the content of a licence file
+     * @param textToShow a message above the licence
+     * @param licenceFile the licence file to display (with 'from java.io import File')
+     * @throws ScriptFailureException
+     * @throws CancelException
+     */
+    public void licence(String textToShow, File licenceFile) throws ScriptFailureException, CancelException {
```
Comment (Tutul-) [inline src/main/java/com/playonlinux/framework/SetupWizard.java]: we must add 'from java.io import File' and use it in the python script like that:
*.licence(text, File("path/to/the/file"))

File: src/main/java/com/playonlinux/framework/SetupWizard.java [hunk 2]
```diff
@@ -103,6 +109,47 @@ public void execute(Message message) {
...
+        }
+    }
+
+    /**
+     * Show a custom licence message
+     * @param textToShow a message above the licence
+     * @param licenceText the licence text to show
+     * @throws CancelException
+     */
+    public void licence(String textToShow, String licenceText) throws CancelException {
```
Comment (Tutul-) [inline src/main/java/com/playonlinux/framework/SetupWizard.java]: and here the (string,string) version, the python script no need modification to use it (more simple to use it that the other)

File: src/main/java/com/playonlinux/framework/SetupWizard.java (hunk 1 above)
Comment (qparis) [inline src/main/java/com/playonlinux/framework/SetupWizard.java]: You need to define three methods: 
- licence(String textToShow, String licenceText)
- licenceFile(String textToShow, File licenceFile)
- licenceFile(String textToShow, String licenceFilePath)

File: src/main/java/com/playonlinux/ui/impl/javafx/setupwindow/StepRepresentationLicence.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import javafx.scene.layout.Pane;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.Text;
+
+import com.playonlinux.common.messages.CancelerMessage;
+import com.playonlinux.common.messages.CancelerSynchroneousMessage;
+
+public class StepRepresentationLicence extends AbstractStepRepresentation {
```
Comment (qparis) [inline src/main/java/com/playonlinux/ui/impl/javafx/setupwindow/StepRepresentationLicence.java]: A licenceStep should not have a leftImage. 
A licenceStep should inheritate at least from AbstractStepRepresentationWithHeader, or even StepRepresentationMessage

**Comment by Tutul-** [comment]
capture d ecran de 2015-06-14 16 16 44

File: src/main/java/com/playonlinux/framework/SetupWizard.java [hunk 4]
```diff
@@ -103,6 +109,54 @@ public void execute(Message message) {
...
     }
+
+    /**
+     * Show the content of a licence file
+     * @param textToShow a message above the licence
+     * @param licenceFile the licence file to display (with 'from java.io import File')
+     * @throws ScriptFailureException
+     * @throws CancelException
+     */
+    public void licenceFile(String textToShow, File licenceFile) throws ScriptFailureException, CancelException {
```
Comment (qparis) [inline src/main/java/com/playonlinux/framework/SetupWizard.java]: No need to throw ScriptFailureException here

File: src/main/java/com/playonlinux/framework/SetupWizard.java [hunk 5]
```diff
@@ -103,6 +109,54 @@ public void execute(Message message) {
...
+    }
+
+    /**
+     * Show the content of a licence file
+     * @param textToShow a message above the licence
+     * @param licenceFilePath the path of the licence file to display
+     * @throws ScriptFailureException
+     * @throws CancelException
+     */
+    public void licenceFile(String textToShow, String licenceFilePath) throws ScriptFailureException, CancelException {
```
Comment (qparis) [inline src/main/java/com/playonlinux/framework/SetupWizard.java]: Same for here

File: src/main/java/com/playonlinux/ui/impl/javafx/setupwindow/StepRepresentationLicence.java [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.TextArea;
+import javafx.scene.text.Text;
+
+import com.playonlinux.common.messages.CancelerMessage;
+import com.playonlinux.common.messages.CancelerSynchroneousMessage;
+
+public class StepRepresentationLicence extends AbstractStepRepresentationWithHeader {
+    String textToShow;
+    String licenceText;
```
Comment (qparis) [inline src/main/java/com/playonlinux/ui/impl/javafx/setupwindow/StepRepresentationLicence.java]: Please set these field private

**Pull request merged**